### DISCUSSION TITLE
Alphabetical ordered list

### DISCUSSION THREAD

**Discussion by Louson** [requarks/wiki#6462] [General]
Hello !

Can we expect support for alphabetical ordered lists ? I know it is not in standard markdown but it is quite useful.

ex:

``` markdown
a. this
a. is
a. an
a. example
```

would result to:

``` text
a. this
b. is
c. an
d. example
```

it is supported in pandoc as fancy_lists

Thanks,
Louis

**Comment by NGPixel** [comment]
Submit a feature request on https://requarks.canny.io/wiki

**Comment by Louson** [comment]
Done https://requarks.canny.io/wiki/p/alphabetical-ordered-list
